### DISCUSSION TITLE
Estagio home office

### DISCUSSION THREAD

**Discussion by Kaedh** [frontendbr/forum#2058] [Perguntas e Respostas]
Estudo análise e desenvolvimento de sistemas, e estou naquela fase em que busco um estágio. como moro no interior queria saber se é possível conseguir uma vaga home office, se tiverem alguma dica de quem já passou por essa etapa seria grato também.

**Comment by antonyleme** [comment]
Cara, possível, é. Mas acho que não é muito comum, principalmente porque estágio está muito relacionado ao aprendizado que você vai adquirir convivendo com pessoas experientes dentro da empresa. Acho muito difícil tu absorver isso no modelo home office porque a gente tende a conviver somente com aqueles que a gente tá diretamente envolvido e online as pessoas tendem a ser menos comunicativas. O que eu vejo por aí e é bem comum, é vaga home office de júnior anunciada como estágio pra pagar menos. Talvez faça sentido pra você. Mas talvez valha a pena você se analisar e ver se pro seu caso não é mais interessante aplicar diretamente pra uma vaga de júnior. Vi no seu perfil que você é de Ribeirão Preto. Cara, pra mim isso é cidade grande. Onde moro tem 70 mil habitantes e consegui boas oportunidades. Talvez você só tenha que procurar melhor. Tenho certeza que devem ter boas oportunidades por aí. No mais, boa sorte.

**Comment by sarahdialmeida** [comment]
Oi Leonardo, eu moro numa zona rural da Bahia e minha única chance de
estagiar nesse período de pandemia seria em um modelo home office. Consegui
uma vaga de estágio recentemente, depois de estudar muito e também de
melhorar a comunicação do meu LinkedIn. Fichei empresas que eu curtia,
aumentei meu engajamento na plataforma e deu certo. É ótima a sua idéia de
perguntar nos fóruns! Continue tentando que a sua vaga vai chegar.

Em dom., 11 de abr. de 2021 às 02:00, Antony Leme ***@***.***>
escreveu:
